uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -909,26 +911,34 @@ sub update_mri_acquisition_dates {
     ############################################################
     # get the registered acquisition date for this session #####
     ############################################################
-    my $query = "SELECT s.ID, m.AcquisitionDate FROM session AS s LEFT OUTER".
-                " JOIN mri_acquisition_dates AS m ON (s.ID=m.SessionID)". 
-                " WHERE s.ID='$sessionID' AND s.Active='Y' AND".
-                " (m.AcquisitionDate > '$acq_date'". 
-                " OR m.AcquisitionDate IS NULL) AND '$acq_date'>0";
-    
+    my $query = "SELECT s.ID, m.AcquisitionDate "
+                . " FROM session AS s "
+                . " LEFT OUTER JOIN mri_acquisition_dates AS m ON (s.ID=m.SessionID)"
+                . " WHERE s.ID=? AND s.Active='Y'";
+    my @values = ($sessionID);
+    if ($acq_date) {
```

Comment:
I think this part of the query should be revisited/simplified to take into account all cases (`$acq_date` defined, `$acq_date` defined but set to `''` or `'0000-00-00'`, `$acq_date` not defined)

Reply:
so I modified the code so that if the date is `'0000-00-00'`, `''` or `0`, `$acq_date` is set to `undef` because if the date is set to one of these values, we don't want to update the table with those values.